CudaArchitectureSelector.cmake

```diff
@@ -1,5 +1,5 @@
 ################################ License #######################################
-# Copyright 2017-2018
+# Copyright 2017-2020
 # 
 # Karlsruhe Institute of Technology Universitat Jaume I University of Tennessee
```

Comment:
Not sure, but maybe we should change this to how it is in the [Ginkgo library](https://github.com/ginkgo-project/ginkgo/blob/develop/benchmark/conversions/conversions.cpp#L2) ?

Reply:
Good point!